products/error_tracking/frontend/scenes/ErrorTrackingFingerprintsScene/errorTrackingIssueFingerprintsSceneLogic.tsx

```diff
@@ -52,27 +56,41 @@ export const errorTrackingIssueFingerprintsSceneLogic = kea<errorTrackingIssueFi
         issue: {
             loadIssue: async () => await api.errorTracking.getIssue(props.id),
         },
-        fingerprints: [
-            [] as ErrorTrackingIssueFingerprint[],
+        issueFingerprints: {
+            loadIssueFingerprints: async () => (await api.errorTracking.fingerprints.list(props.id)).results,
+            split: () =>
+                values.issueFingerprints.filter(
+                    (f: ErrorTrackingFingerprint) => !values.selectedFingerprints.includes(f.fingerprint)
```

Comment:
The type here should be knowable, right?
```suggestion
                    (f) => !values.selectedFingerprints.includes(f.fingerprint)
```

Reply:
Looks it didn't, there was a typecheck error in tests